### ISSUE TITLE
Upgrade Devise Gem

### ISSUE THREAD

**Issue by bsp3ars** [uclibs/ucrate#942]
### Descriptive summary

The devise gem needs to be upgraded to >= 4.7.1

Bundler-audit message:

Name: devise
Version: 4.6.0
CVE: CVE-2019-16109
GHSA: GHSA-fcjw-8rhj-gwwc
Criticality: Medium
URL: heartcombo/devise#5071
Title: Devise Gem for Ruby confirmation token validation with a blank string
Solution: upgrade to >= 4.7.1

**Comment by crowesn** [comment]
```
Bundler could not find compatible versions for gem "devise":
  In Gemfile:
    devise (~> 4.7.1)

    devise-multi_auth was resolved to 0.2.0, which depends on
      devise (= 4.6.0)
```
Gemfile:
`gem 'devise-multi_auth', git: 'https://github.com/uclibs/devise-multi_auth', branch: 'rails-5.1.6.2'`

**Comment by scherztc** [comment]
Closed by JIRA : https://ucdts.atlassian.net/browse/LIBSCHOLAR-39
